sessionctx/variable/varsutil.go

```diff
@@ -156,6 +165,181 @@ func ValidateGetSystemVar(name string, isGlobal bool) error {
 	return nil
 }
 
+// ValidateSetSystemVar checks if system variable satisfies specific restriction.
+func ValidateSetSystemVar(name string, value string) (string, error, error) {
```

Comment:
We may not need to return warn?

Reply:
yes, If we take the `SessionVars` as a parameter into function `ValidateSetSystemVar`, we can directly append warning in this function. I'll fix this